packages/@ember/-internals/glimmer/tests/integration/components/input-angle-test.js

```diff
@@ -280,10 +287,16 @@ moduleFor(
       // this.assertSelectionRange(8, 8); //NOTE: this fails in IE, the range is 0 -> 0 (TEST_SUITE=sauce)
     }
 
-    ['@test sends an action with `<Input @enter={{action "foo"}} />` when <enter> is pressed'](
+    [`${testUnless(
+      DEPRECATIONS.DEPRECATE_TEMPLATE_ACTION.isRemoved
+    )} sends an action with \`<Input @enter={{action "foo"}} />\` when <enter> is pressed`](
```

Comment:
Should this test remain and be converted to calling a function or is that tested elsewhere?

Reply:
I don't think so, the test title was very clear about testing action behavior?

Reply:
At some point actions were synonymous with callbacks from the invoked component, so I didn't know if there were any tests that `@enter={{this.foo}}` would call `this.foo`